WcaOnRails/app/models/registration.rb

```diff
@@ -54,13 +54,7 @@ def gender
   end
 
   def countryId
-    if user
-      country = Country.find_by_iso2(user.country_iso2)
-      if country
-        return country.id
-      end
-    end
-    read_attribute(:countryId)
+    user&.country&.id || read_attribute(:countryId)
```

Comment:
Yeah, IMO we could. I think it's about *personal style*. In cases like this one above it's really awesome construction, so to be consistent we can use it in simpler cases as well =)